bench-tps/src/main.rs

```diff
@@ -145,7 +145,12 @@ fn main() {
     solana_metrics::set_panic_hook("bench-tps", /*version:*/ None);
 
     let matches = cli::build_args(solana_version::version!()).get_matches();
-    let cli_config = cli::extract_args(&matches);
+    let cli_config = cli::parse_args(&matches);
+    if let Err(error) = cli_config {
+        eprintln!("{error}");
+        exit(1);
+    }
+    let cli_config = cli_config.unwrap();
```

Comment:
I think a more idiomatic way to write this would be:
```rust
    let cli_config = match cli::parse_args(&matches) {
        Ok(config) => config,
        Err(error) => {
            eprintln!("{error}");
            exit(1);
        }
    };
```
Name shadowing is also considered good style, if you are transforming the same kind of data in a one way direction.  So, if you want the `cli::parse_args()` call to be a bit more visible, you can also write it like this:
```rust
    let cli_config = cli::parse_args(&matches);

    let cli_config = match cli_config {
        Ok(config) => config,
        Err(error) => {
            eprintln!("{error}");
            exit(1);
        }
    };
```
I would personally go with the first option, as this is quite common.
And generally people should be ready to read it with no problem.